yarn.lock

```diff
@@ -11956,7 +11956,7 @@ web3-core-subscriptions@1.3.4:
     underscore "1.9.1"
     web3-core-helpers "1.3.4"
 
-web3-core@1.3.4, web3-core@^1.3.0:
+web3-core@1.3.4:
```

Comment:
it's still installed 😛

Reply:
yeah, as it some of the packages still have the older version installed and I can only completely update that once everything is on npm